frontend/src/metabase/visualizations/lib/settings/series.js

```diff
@@ -14,14 +14,14 @@ export function keyForSingleSeries(single) {
 
 const LINE_DISPLAY_TYPES = new Set(["line", "area"]);
 
+export const settingId = "series_settings";
+export const colorSettingId = "series_settings.colors";
```

Comment:
nit: since this is outside they're now global variables. Should we change the name to [SCRAEMING_SNAKE_CASE](https://metaboat.slack.com/archives/C505ZNNH4/p1668013402947669)? 🐍